persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/conversation/ConversationDAOImpl.kt

```diff
@@ -133,11 +133,11 @@ internal class ConversationDAOImpl internal constructor(
         }
     }
 
-    override suspend fun insertOrIgnoreConversations(conversationEntities: List<ConversationEntity>) = withContext(coroutineContext) {
+    override suspend fun insertOrUpdateLastModified(conversationEntities: List<ConversationEntity>) = withContext(coroutineContext) {
```

Comment:
It would be nice to cover this function with a couple of tests.
Making sure it only updates the `lastModifiedDate`, does not overwrite any other data, etc.